.circleci/config.yml

```diff
@@ -0,0 +1,37 @@
+# Python CircleCI 2.0 configuration file
+# Updating CircleCI configuration from v1 to v2
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2
+jobs:
+
+  test:
```

Comment:
I was wrong. We do need a build job here. Or we need to use a custom [workflow](https://circleci.com/docs/2.0/workflows/).